src/leiningen/ezbake.clj

```diff
@@ -12,6 +12,7 @@ as defined below.
 Actions:
   stage      Generate and stage ezbake artifacts.
   build      Build native packages from staged artifacts.
+  json       Generate ezbake manifest json file.
```

Comment:
I would probably do with `manifest` or maybe `deps-manifest` if we wanted to be explicit.